src/transformers/utils/py_utils.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python utilities"""
+
+from collections.abc import MutableMapping
+
+
+def flatten_dict(d: MutableMapping, parent_key: str = "", delimiter: str = "."):
```

Comment:
This should go in utils.generic instead, not need to create a new file for this IMO

Reply:
Will do